bundles/org.openhab.binding.evohome/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -4,7 +4,7 @@
 		<supported-bridge-type-refs>
 			<bridge-type-ref id="account" />
 		</supported-bridge-type-refs>
-		<label>evohome Display</label>
```

Comment:
In this case its a product name, however, they are not too good anyway in promoting the correct usage

Reply:
Yes, I remember this from the review - it should be kept lower case as the product name.